core/src/main/scala/cromwell/core/ConfigUtil.scala

```diff
@@ -7,15 +7,15 @@ import cats.syntax.validated._
 import com.typesafe.config.{Config, ConfigException, ConfigValue}
 import org.slf4j.LoggerFactory
 
-import scala.collection.JavaConversions._
+import scala.collection.JavaConverters._
 import scala.reflect.{ClassTag, classTag}
 
 object ConfigUtil {
 
   val validationLogger = LoggerFactory.getLogger("ConfigurationValidation")
 
   implicit class EnhancedConfig(val config: Config) extends AnyVal {
-    def keys = config.entrySet().toSet map { v: java.util.Map.Entry[String, ConfigValue] => v.getKey }
+    def keys = config.entrySet().asScala.toSet map { v: java.util.Map.Entry[String, ConfigValue] => v.getKey }
```

Comment:
Oh I think I see... the `config.entrySet` is a (pardon the syntax) `JavaSet[JavaMap.Entry]` and `asScala` makes it a `ScalaSet[JavaMap.Entry]`. LGTM